assets/dns/daemonset.yaml

```diff
@@ -33,7 +35,7 @@ spec:
             port: 8181
             scheme: HTTP
           initialDelaySeconds: 10
-          periodSeconds: 3
+          periodSeconds: 3 # Update minReadySeconds if you change this value!
```

Comment:
```suggestion
          periodSeconds: 3 # Update the daemonset spec.minReadySeconds if you change this value!
```

Reply:
Updated in https://github.com/openshift/cluster-dns-operator/compare/b0ca398b84de1287fe02d22bf9311f0053a02d1c..043594d697605804dae0952b77c741270c0042bc.